tools/unix/generate_localizations.sh

```diff
@@ -66,4 +66,12 @@ cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/countries_names.txt" "$OMIM_PATH/data/countries-strings/"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/strings/sound.txt" "$OMIM_PATH/data/sound-strings/"
 
+# Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
+SUPPORTED_LOCALIZATIONS=$(sed -nE "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1,/p" "${OMIM_PATH}/data/strings/strings.txt" | sort -u | tr '\n' ' ' | sed -E 's/, $//')
+SUPPORTED_LOCALIZATIONS=$(echo listLanguages="$SUPPORTED_LOCALIZATIONS" | sed 's/[[:space:]]//g' | tr '-' '_')
```

Comment:
You definitely can generate the necessary list format in the above command, instead of fixing it here. Or no?

Reply:
Unix tell me too many arguments

Reply:
Fixed :)